apps/package.json

```diff
@@ -62,6 +62,7 @@
     "babel-loader": "7.1.5",
     "babel-plugin-add-module-exports": "^0.2.1",
     "babel-plugin-syntax-async-functions": "^6.8.0",
+    "babel-plugin-syntax-dynamic-import": "^6.18.0",
```

Comment:
Have you tested this in IE11? [The docs for this plugin](https://babeljs.io/docs/en/babel-plugin-syntax-dynamic-import) mention that IE requires polyfills for promise and iterator for this to work.  I know we polyfill promise everywhere, I can't remember if we polyfill iterator.